packages/api/prisma/migrations/20210623204743_user/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- AlterTable
```

Comment:
I'm not suer if migrations is right tool, if the app is not on prod. 
But I don't know best practices with prisma.

Reply:
I will check it out